### DISCUSSION TITLE
matr_019 - Sesso testimone

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#952] [Supporto servizi cooperativi] [answered]
Buongiorno,
nella versione 1.30.0 il campo sesso testimone è divenuto obbligatorio solo per il secondo, terzo e quarto testimone.
Nel primo il campo risulta facoltativo.
Immagino che siano tutti e 4 obbligatori.
Ho trovato lo stesso problema anche in altri usecase

**Comment by amusarra** [comment, marked as answer]
Salve @annamaraspin 
Abbiamo verificato che si tratta di un bug; trasformiamo di conseguenza questa discussione in issue.

Cordiali Saluti.
Supporto ANSC.

**Reply by amusarra** [reply]
La issue è #957
